plugins/arElasticSearchPlugin/lib/arElasticSearchPlugin.class.php

```diff
@@ -605,9 +680,9 @@ private function displayTypesToIndex($excludeTypes)
 
         $this->log('Types that will be indexed:');
 
-        foreach ($this->mappings as $typeName => $typeProperties) {
-            if (!in_array(strtolower($typeName), $excludeTypes)) {
-                $this->log(' - '.$typeName);
+        foreach ($this->mappings as $indexName => $indexProperties) {
+            if (!in_array(strtolower($indexName), $excludeTypes)) {
+                $this->log(' - '.$indexName);
                 ++$typeCount;
             }
         }
```

Comment:
Okay, maybe not here. These need to match the `exclude-types` argument from the search:populate task.

Now that I think about that, we should probably revisit how is that done with multiple indexes instead of types.